vimeo-networking/src/main/java/com/vimeo/networking/model/VimeoAccount.java

```diff
@@ -95,13 +108,12 @@ public User getUser() {
         return this.user;
     }
 
-    public void setUser(User user) {
+    public void setUser(@Nullable User user) {
         this.user = user;
     }
 
     @Nullable
-    public String getUserJSON() // For AccountManager.userData [AH]
-    {
+    public String getUserJSON() {
         if (this.user == null) {
             return null;
         }
```

Comment:
👍 nice call out - it wasn't something I was paying attention to, but I will make sure it gets done right as part of this PR.